app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -0,0 +1,120 @@
+/* ********* Imports: ********* */
+package com.example.sabonit;
+import android.content.Context;
+import android.telephony.TelephonyManager;
+
+
+/**
+ * This class represents a user's account, which holds all the data of the user. The operations of
+ * the users with the database are based on the data stored in the account. Therefore the attribute
+ * phoneNumber needs to be unique, as a primary key. If a phone number stored in the google account
+ * and no phone number given as an input to the constructor, the phone number in the google account
+ * will be taken.
+ */
+public class Account
+{
+    /* ********* Constants: ********* */
+    // For validity phone number checking
+    private static int IMPLICIT_ISRAELI_PHONE_NUMBER_LENGTH = 9;
+    private static int ISRAELI_AREA_CODE = 13;
+    private static String ISRAELI_AREA_CODE_INFIX = "+972";
+
+
+    /* ********* Attributes: ********* */
+    // Given as an address
+    String location;
+    // User's name, may can be extracted from the google acount
+    String name;
+    // User's phone number, may can be extracted from the google acount
+    String phoneNumber;
+    // User's cart, empty when initialized
+    Cart cart;
```

Comment:
it's an object?

Reply:
Yes, I gonna put the next code pieces of Cart and Order in this morning